### ISSUE TITLE
test(windows):Regression Test Report 17.0.196-Alpha (4failures)

### ISSUE THREAD

**Issue by sillsdev-dondillard** [keymanapp/keyman#9881]
```
Test Report for Keyman 17.0.196 Alpha Windows
Results 

 Expected Start Date 10/24/2023
 Actual Start Date 10/24/2023

 Overall Results: 94% (64/68 passed)

Regression Tests
 Windows 10 94% (32/34 passed)
 Windows 11 94% (32/34 passed)

Issues Found
```
#9840 
#9848

**Comment by mcdurdin** [comment]
> #9480

This pull request number seems wrong?

**Comment by mcdurdin** [comment]
> #9848

And this relates to a Keyman Developer issue, not a Keyman for Windows issue.

**Comment by sillsdev-dondillard** [comment]
I transposed the numbers it should be #9840. I updated the report.

 

Don Dillard

LTDev Software Test Manager

  <https://ci3.googleusercontent.com/mail-sig/AIorK4y-D6efUUoX-1TX63MeJdjG_TLa0Q0pcXVM1MIKn_Avt5EppfuTZXfKyTuNCi7_u9VW-QbJ0UA> 

SIL International

7500 West Camp Wisdom Rd

Dallas TX 75326

425-273-1965

 ***@***.***> ***@***.***

 

“I am the Vine; you are the branches. The one who remains in me and I in him is the one who bears much fruit, because without me you can do nothing.”

John 15:5  EHV

**Comment by sillsdev-dondillard** [comment]
Updated transposed numbers on an issue changed from 9480 to 9840

**Comment by mcdurdin** [comment]
Thanks for the updated number. Question on #9848 -- this is a Keyman Developer issue, so not sure how it comes up on a Keyman for Windows regression test?

**Comment by rc-swag** [comment]
Issue #9840 is closed and #9848 (even with it being a developer issue is closed) - Close this issue.
